Projects/surface_photometry.ipynb

```diff
@@ -0,0 +1,1332 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Doing quick surface photometry of a galaxy given its coordinates"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Owners: **Aaron Watkins, Nushkia Chamba**  \n",
```

Comment:
Do you want to add a link to your github user accounts (so people can get in touch with you)?

Reply:
Sure, go right ahead.